main.go

```diff
@@ -484,6 +484,7 @@ func mainCore() error {
 	webMux.Mount("/explorer", explore.Mux)
 	webMux.Get("/blocks", explore.Blocks)
 	webMux.Get("/mempool", explore.Mempool)
+	webMux.With(explorer.AddressRowIDCtx).Get("/tx/addressid/{id}", explore.TxByAddressRowID)
```

Comment:
This reason why I choose to use `/tx/addressid/{id}` instead of `/tx/{txid}` is because we are using `in_out_row_id` value that is stored in the database to match one tx to another. `in_out_row_id` is the row id of the matching tx. I initial thought of using `in_out_row_id` to fetch the `tx_hash` (in the respective queries) or simply the `txid` that can be used with `/tx/{txid}` but it felt that reusing `/tx/{txid}` would be very inefficient and reusing what is readily available from the database would have been better (`in_out_row_id`).